core/option/options.go

```diff
@@ -45,6 +45,13 @@ type CountOptioner interface {
 	countOption()
 }
 
+// EstimatedDocumentCountOptioner is the interface implemented by types that can be used as
+// Options for EstimatedDocumentCount command.
+type EstimatedDocumentCountOptioner interface {
```

Comment:
It doesn't seem like we need this type at all, it isn't used in core and only used slightly in mongo.